Makefile

```diff
@@ -853,6 +853,11 @@ bench: bench-net bench-http bench-fs bench-tls
 
 bench-ci: bench
 
+mdlint:
+	@echo "Running Markdown linter..."
+	$(NODE) tools/remark-cli/cli.js -q \
+	  doc/api doc/api_assets doc/guides src lib benchmark tools/doc/ tools/icu/
```

Comment:
Looking at this [comment](https://github.com/nodejs/nodejs.org/pull/1130#issuecomment-277912697), I thought that anyone should not touch the commit message in the changelog anyway, but it's fine if we've polished it in the past because it's absolutely better for users who read the rendered docs. Will update it :)